windows.md

```diff
@@ -54,15 +54,60 @@ With those compatibility things out of the way, you're ready to start the system
    ```bash
    choco install slack -y
    ```
-7. Copy the following text and right-click in the blue middle part of the PowerShell window to paste the text. Hit enter.<br><br>
+7. Close PowerShell and open it again as administrator (like in step 2). Copy the following text and right-click in the blue middle part of the PowerShell window to paste the text. Hit enter.<br><br>
    ```bash
    corepack enable
    corepack prepare pnpm@latest --activate
    Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
    pnpm setup
    ```
    This uses Corepack to install `pnpm`, and configures `pnpm`'s global bin directory.<br><br>
-8. Close PowerShell and open it again as administrator (like in step 2). Copy each line in the following text, right-click in the blue middle part of the PowerShell window to paste the text and hit enter.
+   Install `@upleveled/preflight`, a program we will use in the course, to verify that the previous commands were successful: copy the following text, paste it in the terminal and hit return.<br><br>
+
+   ```bash
+   pnpm add --global @upleveled/preflight
+   ```
```

Comment:
it looks like this removal of the "Close PowerShell and open it again as administrator ..." may be introducing a new error:

in [this comment](https://github.com/upleveled/system-setup/issues/41#issuecomment-2082303483) it appears that `pnpm setup` does indeed set environment variables on the system and needs to have a new terminal started (see the bottom of the screenshot)

![Screenshot 2024-04-29 104040](https://github.com/upleveled/system-setup/assets/1935696/2d28836a-b252-44d8-a513-52349321aec4)

Reply:
I successfully uninstalled Node and removed all the Windows environment variables used by pnpm. Reinstalled Node with `choco` and followed the System Setup steps.

In the screenshot, pnpm confirms that the setup is complete with the message 

```bash
Setup complete. Open a new terminal to start using pnpm.
```

![pnpm-install](https://github.com/user-attachments/assets/124dde2c-3dab-46c6-a1b4-f7e0d0dd77f1)
At the bottom of the screenshot, we can see that `pnpm` logs `Setup complete. Open a new terminal to start using pnpm`

I added the `Close PowerShell and open it again as administrator (like in step 2).` to the instructions in this commit
[43408f5](https://github.com/upleveled/system-setup/pull/91/commits/43408f5dcd8891b07e46794de0687a52d20ce962)